modules/installation-configuration-parameters.adoc

```diff
@@ -491,7 +516,12 @@ endif::vsphere[]
 ifdef::ibm-power-vs[]
 Example usage, `compute.platform.powervs.sysType`.
 endif::ibm-power-vs[]
+ifndef::agent[]
 |`alibabacloud`, `aws`, `azure`, `gcp`, `ibmcloud`, `nutanix`, `openstack`, `powervs`, `vsphere`, or `{}`
+endif::agent[]
+ifdef::agent[]
+|`vsphere`
```

Comment:
@mhanss do I need to add `{}` and/or `external` to this parameter?

Reply:
@skopacz1 You can set `controlPlane.Platform` and `compute.Platform` to `{}` for agent installer like below:
```
            "controlPlane": {
                "name": "master",
                "replicas": 1,
                "platform": {},
                "hyperthreading": "Enabled",
                "architecture": "amd64"
            },
            "compute": [
                {
                    "name": "worker",
                    "replicas": 0,
                    "platform": {},
                    "hyperthreading": "Enabled",
                    "architecture": "amd64"
                }
            ],
```
I don't think you need to specify other possible platform values in there. The possible values could be ( no external here),
```
"controlPlane": {
                "name": "master",
                "replicas": 1,
                "platform": {
                    "aws": {
                        "zones": ["us-east", "us-west"],
                        "type": "test"
                    }
                },
                "hyperthreading": "Enabled",
                "architecture": "amd64"
            },
```
Here is the schema definition for `platform` section for controlPlane and compute https://github.com/openshift/installer/blob/master/pkg/types/machinepools.go#L64

Each of the underlying platform schema definition is here https://github.com/openshift/installer/blob/master/pkg/types/machinepools.go#L84 and any specific ones are here https://github.com/openshift/installer/blob/master/pkg/types/aws/machinepool.go#L5



Long story short, for agent installer, you can just set `controlPlane.Platform` and `compute.Platform`  to `{}`

Reply:
Thanks for the detailed response, Pawan!